packages/database/test/query.test.ts

```diff
@@ -15,11 +15,15 @@
  * limitations under the License.
  */
 
-import { expect } from 'chai';
+import * as chai from 'chai';
+const expect = chai.expect;
```

Comment:
Can you clean up the order here and move this to the bottom:

```
const expect = chai.expect;
chai.use(chaiAsPromised);
```

You could also just do:

```
import {use, expect} from 'chai';
```